Add/universal login (#9038)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/about-cloud/regions-ip-addresses.md

## Accessing your account

> The recommended way to sign in is [https://login.dbt.com](https://login.dbt.com). Enter your email, verify it, and select the account you want to open from the list of accounts associated with your email. For more information about the login process, refer to [Log in to dbt platform](/docs/cloud/about-cloud/login).

Author: ```suggestion
The recommended way to sign in to <Constant name="dbt_platform"/> is [https://login.dbt.com](https://login.dbt.com). Enter your email, verify it, and select the account you want to open from the list of accounts associated with your email. For more information about the login process, refer to [Log in to dbt platform](/docs/cloud/about-cloud/login).
```

Reviewer: Constant variable query here - maybe use: <Constant name="dbt_platform" />

- **APAC multi-tenant:** `ACCOUNT_PREFIX.au1.dbt.com` (for example, `abc123.au1.dbt.com`)
- **Worldwide single-tenant and VPC:** Use the vanity URL provided during your onboarding.

Refer to the [tables earlier](#AWS) for the full list of access URLs by region.

## Locating your dbt IP addresses

There are two ways to view your <Constant name="dbt_platform" /> IP addresses:
- If no projects exist in the account, create a new project, and the IP addresses will be displayed during the **Configure your environment** steps.
- If you have an existing project, navigate to **Account Settings** and ensure you are in the **Projects** pane. Click on a project name, and the **Project Settings** window will open. Locate the **Connection** field and click on the name. Scroll down to the **Settings**, and the first text block lists your IP addresses. 

### Static IP addresses

<Constant name="dbt_platform" /> is hosted on AWS, Azure, and the Google Cloud Platform (GCP). While we can offer static URLs for access, we cannot provide a list of IP addresses to configure connections due to the nature of these cloud services.

> * Dynamic IP addresses &mdash; <Constant name="dbt_platform" /> offers static URLs for streamlined access, but the dynamic nature of cloud services means the underlying IP addresses change occasionally. The cloud service provider manages the IP ranges and may change them according to their operational and security needs.

Reviewer: ```suggestion
* Dynamic IP addresses &mdash; <Constant name="dbt_platform" /> provides static URLs for streamlined access, but the dynamic nature of cloud services means the underlying IP addresses can change occasionally. Cloud providers manage these IP ranges and may update them based on operational and security needs.
```

> * Using hostnames for consistent access &mdash; To ensure uninterrupted access, we recommend that you use <Constant name="dbt_platform" /> services using hostnames. Hostnames provide a consistent reference point like `abc123.us1.dbt.com`, regardless of any changes in underlying IP addresses. We are aligning with an industry-standard practice employed by organizations such as Snowflake.

Reviewer: ```suggestion
* Using hostnames for consistent access &mdash; To ensure uninterrupted access, use <Constant name="dbt_platform" /> hostnames. Hostnames provide a consistent reference point, such as `abc123.us1.dbt.com`, even if underlying IP addresses change. This aligns with an industry-standard practice used by platforms such as Snowflake.
```

* Optimizing VPN connections &mdash; You should integrate a proxy alongside VPN for users who leverage VPN connections. This strategy enables steady IP addresses for your connections, facilitating smooth traffic flow through the VPN and onward to <Constant name="dbt_platform" />. By employing a proxy and a VPN, you can direct traffic through the VPN and then to <Constant name="dbt_platform" />. It's crucial to set up the proxy if you need to integrate with additional services.

## API Access URLs

> <Constant name="dbt_platform" /> accounts with cell-based account prefixes have unique access URLs for account APIs. These URLs can be found in your **Account settings** below the **Account information** pane.

Reviewer: ```suggestion
For <Constant name="dbt_platform" /> accounts with cell-based account prefixes, account API access URLs are unique per account. You can find these URLs in **Account settings**, under **Account information**.
```
